agr_java_core/src/main/java/org/alliancegenome/cache/repository/helper/AlleleFiltering.java

```diff
@@ -76,13 +76,22 @@
 
 
     private static FilterFunction<Allele, String> molecularConsequenceFilter =
-            (allele, value) ->
-                    FilterFunction.fullMatchMultiValueOR(allele.getVariants()!=null && allele.getVariants().size()>0?allele.getVariants().stream()
-                            .filter(Objects::nonNull)
-                            .filter(variant -> variant.getGeneLevelConsequence() != null)
-                            .map(variant -> variant.getGeneLevelConsequence().getGeneLevelConsequence())
-                            .collect(Collectors.toSet()):new HashSet<>(), value);
+            (allele, value) -> {
+
+              if(  allele.getVariants()==null || allele.getVariants().size()==0)
+                  return false;
+              else {
+               Set<String> molecualarConsequences=  allele.getVariants().stream()
```

Comment:
please change variable name to 
molecularConsequences

Reply:
@cmpich fixed typo.